src/mugs.js

```diff
@@ -14,6 +14,235 @@ define([
     util,
     undefined
 ) {
+    /**
+     * A question, containing data, bind, and control elements.
+     */
+    function Mug(options, form, baseSpec, copyFromMug) {
```

Comment:
This (most important) class is now at the top of the file where I can easily find it.